locales/lt_LT/LC_MESSAGES/messages.po

```diff
@@ -17,7 +17,7 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: lt\n"
-"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+"Plural-Forms: nplurals=4; plural=(n % 10 == 1 && (n % 100 > 19 || n % 100 < 11) ? 0 : (n % 10 >= 2 && n % 10 <=9) && (n % 100 > 19 || n % 100 < 11) ? 1 : n % 1 != 0 ? 2: 3);\n"
```

Comment:
This `Plural-Forms` string was exported by Transifex, I did not check it in detail. Transifex changed the plural forms of some slavic languages some month ago and added one more plural form to conform to [Unicode CLDR](http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html).

The new rules seem to be equal to the [rules for Lithuanian](http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#lt).

The check `n % 1 != 0` seems to be necessary because decimal numbers have an own plural form. I think that's what they express with `f != 0`.

Reply:
I removed the plural forms update.